crypto-volatility-index/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "crypto-volatility-index",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.1",
```

Just to match the versioning convention of our other adapters.